The code here uses ifunc which we don't have:

../../../../../src/3rdparty/chromium/base/allocator/partition_allocator/tagging.cc:17:10: fatal error: 'sys/ifunc.h' file not found
   17 | #include <sys/ifunc.h>

--- a/src/3rdparty/chromium/base/allocator/partition_allocator/partition_alloc.gni
+++ b/src/3rdparty/chromium/base/allocator/partition_allocator/partition_alloc.gni
@@ -29,8 +29,7 @@
   assert(false, "Unknown CPU: $current_cpu")
 }
 
-has_memory_tagging =
-    current_cpu == "arm64" && is_clang && !is_asan && (is_linux || is_android)
+has_memory_tagging = false
 
 declare_args() {
   # Causes all the allocations to be routed via allocator_shim.cc. Usually,
